### PR TITLE
feat(edge-local-hooks): CEXT-2904 - Updated hooks plugin version.

### PR DESCRIPTION
**Pull request by kmaschi**
## Description

Bumped hooks plugin version.

## Related Issue

CEXT-2904

## Motivation and Context

Allows for ES6+ imports to support additional runtime environments, such as edge compute.

## How Has This Been Tested?

- Create a new local project
  - `aio api-mesh init`
- Run local development mesh
  - `aio api-mesh run mesh.json`

## Screenshots (if appropriate):

image

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by dthampy** [comment]
Would it make sense to consider publishing a beta version with this change?

**Comment by kmaschi** [comment]
> Would it make sense to consider publishing a beta version with this change?

Updated to alpha release tag based on alpha/beta package versions of mesh-builder and plugin-hooks.

**Comment by amolina-adobe** [comment]
> Updated to alpha release tag based on alpha/beta package versions of mesh-builder and plugin-hooks.

Do you intend to publish the `alpha` version of the CLI?  If so, merging this PR to the `develop` branch won't publish anything as you can only publish a `beta` version from develop.

**Comment by kmaschi** [comment]
> > Updated to alpha release tag based on alpha/beta package versions of mesh-builder and plugin-hooks.
> 
> Do you intend to publish the `alpha` version of the CLI? If so, merging this PR to the `develop` branch won't publish anything as you can only publish a `beta` version from develop.

Updated to beta version.

**Review by amolina-adobe — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -37,13 +37,13 @@
...
-		"@adobe-apimesh/mesh-builder": "1.6.0",
+		"@adobe-apimesh/mesh-builder": "1.6.2-alpha.3",
 		"@adobe/aio-cli-lib-console": "^5.0.0",
 		"@adobe/aio-lib-core-config": "^5.0.0",
 		"@adobe/aio-lib-core-logging": "^3.0.0",
 		"@adobe/aio-lib-env": "^3.0.0",
 		"@adobe/aio-lib-ims": "^7.0.1",
-		"@adobe/plugin-hooks": "0.3.0",
+		"@adobe/plugin-hooks": "0.3.2-beta",
 		"@adobe/plugin-on-fetch": "0.1.0",
```
Comment (dthampy) [inline package.json:47]: @kmaschi The latest package version for plugin-on-fetch shows `0.1.1-alpha` there in that [repo](https://github.com/adobe/plugin-on-fetch/pull/8/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R3)
So wondering if the on-fetch plugin needs to be released as a full version and reference  that here ?

Comment (kmaschi) [reply package.json]: I've updated this version to match what is currently in the alpha mesh builder. Speaking with the team it sounds like those changes are still being tested. The intent of this beta branch is to verify mesh-builder functionality and at least the plugin-hooks changes.

**Review by dthampy — APPROVED** [review]

**Review by amolina-adobe — APPROVED** [review]

File: src/templates/package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
 		"@graphql-mesh/soap": "0.14.25",
 		"@graphql-mesh/http": "^0.96.9",
 		"graphql": "^16.6.0",
-		"@adobe/plugin-hooks": "0.3.0",
+		"@adobe/plugin-hooks": "0.3.2-beta",
 		"@adobe/plugin-on-fetch": "^0.1.0",
```
Comment (dthampy) [inline src/templates/package.json:30]: @kmaschi Doesn't the version here need to be updated as well to `0.1.1-alpha`?

Comment (kmaschi) [reply src/templates/package.json]: It has been bumped here as well. Thanks for catching this.

**Review by dthampy — APPROVED** [review]

**Review by amolina-adobe — APPROVED** [review]

**Pull request merged**